userspace/libsinsp/filterchecks.cpp

```diff
@@ -2361,7 +2361,7 @@ const filtercheck_field_info sinsp_filter_check_thread_fields[] =
 	{PT_CHARBUF, EPF_NONE, PF_NA, "proc.env", "Environment", "The environment variables of the process generating the event."},
 	{PT_CHARBUF, EPF_NONE, PF_NA, "proc.cwd", "Current Working Directory", "The current working directory of the event."},
 	{PT_INT64, EPF_NONE, PF_ID, "proc.loginshellid", "Login Shell ID", "The pid of the oldest shell among the ancestors of the current process, if there is one. This field can be used to separate different user sessions, and is useful in conjunction with chisels like spy_user."},
-	{PT_INT32, EPF_NONE, PF_ID, "proc.tty", "Process TTY", "The controlling terminal of the process. 0 for processes without a terminal."},
+	{PT_UINT32, EPF_NONE, PF_ID, "proc.tty", "Process TTY", "The controlling terminal of the process. 0 for processes without a terminal."},
```

Comment:
> 0 for processes without a terminal.

But in gvisor (and possibly elsewhere) we are using `UINT32_MAX` now. Can we either fix gvisor (ie: take it back to use `0` for tty, or fix this description?

Reply:
@FedeDP yes this should be 0, also found one more overlooked int32_t -> uint32_t for scap file size check, just so we ensure maybe now we have sanity and consistency, fingers crossed.

Reply:
Thank you!!